### ISSUE TITLE
open_db syntax and parameters

### ISSUE THREAD

**Issue by aracnus** [Dainanahan/dbparser#55]
How exactly open_db works? I used the default syntax as described at documentation ("Saving into a database" section), changing to my settings, like this:

`open_db(xml_db_name =  "drugbank.xml", driver = "MariaDB", server = "127.0.0.1", output_database = "drugbank")`

And got this error:

`Error in open_db(xml_db_name = "drugbank.xml", driver = "MariaDB", server = "127.0.0.1",  : 
  unused argument (xml_db_name = "drugbank.xml")`

suggesting that "xml_db_name" isn't an acceptable parameter.

To test my connection, I remove this parameter and now I'm having this error:

`Erro: nanodbc/nanodbc.cpp:950: 28000: [ma-3.1.2]Access denied for user 'UNKNOWN_USER'@'localhost' (using password: NO)`

Since the connection is being made via ODBC (is that right?), I configured my odbc.ini and odbcinst.ini, but there is no parameter to indicate my connection settings (available at odbc.ini). So, how can I indicate user and password to this connection?

I'm using R, version 3.6.1 on GNU/Linux.

**Comment by sharmb** [comment]
I had a similar problem, removing the xml_db_name parameter worked for me. 

Perhaps use another driver name, such as "SQL Server"

**Comment by MohammedFCIS** [comment]
@aracnus thank you for submitting the issue, it will be reviewed and fixed (if needed) so soon.

**Comment by MohammedFCIS** [comment]
@aracnus -  MariaDB is now supported and it will be available on CRAN soon.
Kindly find the following tutorials for MariaDB, I found them useful
- https://mariadb.com/kb/en/library/r-statistical-programming-using-mariadb-as-the-background-database/#using-r-with-mariadb
- https://mariadb.com/kb/en/library/rmariadb/

On the other hand, while I use this db I noticed that its default table size is small in contrast to SQL Server which make it not supporting all drugs framework. This is just fyi.

Thanks again for using dbparser
